### PR TITLE
Migrated test_verify_df_output

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/glusterd/test_verify_df_output.py)

Updates: #292

Fixes: #756

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
@srijan-sivakumar @nik-redhat I have added the logic. Lemme know if something has to be changed.

File: common/ops/gluster_ops/brick_ops.py [hunk 1]
```diff
@@ -203,17 +208,17 @@ def replace_brick_from_volume(self, volname: str, node: str,
...
         subvols_list = self.get_subvols(volname, node)
 
+        if not src_brick:
+            # Randomly select a brick to replace
+            src_brick = (random.choice(random.choice(subvols_list)))
         if not dst_brick:
             _, dst_brick = self.form_brick_cmd(servers, brick_roots,
                                                volname, 1, True)
 
+            dst_brick = f"{src_brick.split(':')[0]}:{dst_brick.split(':')[1]}"
```
Comment (nik-redhat) [inline common/ops/gluster_ops/brick_ops.py:218]: Why this change?
This means it becomes necessary to use the same node for the new brick, but why?

Comment (aujjwal-redhat) [reply common/ops/gluster_ops/brick_ops.py:218]: The issue is before replace brick from volume the bricks were :

servera: [/path/to/brick/brick-0], serverb: [/path/to/brick/brick-1], serverc: [/path/to/brick/brick-1]

After replace it came out as :
servera: [/path/to/brick/brick-3, /path/to/brick/brick-4], serverb: [/path/to/brick/brick-5], serverc: []

And in this case df -h gave 4GB instead of 8GB normally...

I tested the same manually as shared in the redant grp

But when it is properly distributed it gave 8GB ... that's why I am considering that if in case dst brick is not given it's better to replace with a brick on the same server...

Comment (srijan-sivakumar) [reply common/ops/gluster_ops/brick_ops.py:218]: I think this might not work with a real scenario. For instance, if we had had a disk for each brick, we would not face this issue. Hence I feel we shouldn't restrict the replace to the same node.

Comment (aujjwal-redhat) [reply common/ops/gluster_ops/brick_ops.py:218]: Yeah .. true

**Comment by nik-redhat** [comment]
Closing this wrt #929
